include/z64actor.h

```diff
@@ -243,7 +243,11 @@ typedef struct Player {
     /* 0x03AC */ Actor*     heldActor;
     /* 0x03B0 */ char       unk_3B0[0x018];
     /* 0x03C8 */ Vec3f      unk_3C8;
-    /* 0x03D4 */ char       unk_3D4[0x060];
+    /* 0x03D4 */ char       unk_3D4[0x058];
+    /* 0x042C */ u8         unk_42C;
+    /* 0x042D */ s8         unk_42D;
+    /* 0x042E */ u16        unk_42E;
+    /* 0x0430 */ Actor*     unk_430; // pointer to DoorGerudo
```

Comment:
This is not necessarily DoorGerudo, it can be other "door" actors. Also I'm already using a custom type for this in my player decomp, so you probably don't need to worry about making a note of it.
```suggestion
    /* 0x0430 */ Actor*     unk_430;
```

Reply:
Ah okay, yeah I wasn't sure so I just left the comment, but happy to get rid of it.